RefManager.py

```diff
@@ -10,8 +10,6 @@
 import os.path, re, lxml, time
 import arelle.ModelDocument
 from arelle.FileSource import openFileSource
-from arelle import PythonUtil # define 2.x or 3.x string types
-PythonUtil.noop(0) # Get rid of warning on PythonUtil import

```

Comment:
This was to bring in the Arelle defined custom builtins for python 2/3 support, but they aren't used anymore.